src/libre/parser.act

```diff
@@ -102,6 +115,14 @@
 	typedef struct fsm * t_fsm;
```

Comment:
Can `t_fsm` go too? and perhaps `struct group` and `t_grp`.

Reply:
I think we should be left with no FSM stuff in the parsers.

Reply:
Indeed. Removed in f5adce5 .